packages/cli/src/options/paramsOptions.ts

```diff
@@ -35,38 +30,3 @@ const paramsOptionsByName = ObjectKeys(chainConfigTypes).reduce(
   },
   {}
 );
-
-const terminalArgsToParamsMap: {[K in keyof ITerminalPowArgs]: keyof ChainConfig} = {
-  "terminal-total-difficulty-override": "TERMINAL_TOTAL_DIFFICULTY",
-  "terminal-block-hash-override": "TERMINAL_BLOCK_HASH",
-  "terminal-block-hash-epoch-override": "TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH",
-};
-
-export function parseTerminalPowArgs(args: ITerminalPowArgs): IBeaconParamsUnparsed {
-  const parsedArgs = ObjectKeys(terminalArgsToParamsMap).reduce((beaconParams: Partial<IBeaconParamsUnparsed>, key) => {
-    const paramOption = terminalArgsToParamsMap[key];
-    const value = args[key];
-    if (paramOption != null && value != null) beaconParams[paramOption] = value;
-    return beaconParams;
-  }, {});
-  return parsedArgs;
-}
-
-export const paramsOptions: CliCommandOptions<IParamsArgs> = {
-  ...paramsOptionsByName,
-
-  "terminal-total-difficulty-override": {
```

Comment:
I don't expect anyone is still setting these flags, but we wanna be extra careful could keep them around